hybridauth/Hybrid/Logger.php

```diff
@@ -77,7 +77,7 @@ public static function info($message) {
 	 */
 	public static function error($message, $object = null) {
 		if (isset(Hybrid_Auth::$config["debug_mode"]) && in_array(Hybrid_Auth::$config["debug_mode"], array(true, 'info', 'error'), true)) {
-                        $dt = new DateTime;
+                        $dt = new DateTime('now',  new DateTimeZone( 'UTC' ));$dt = new DateTime;

```

Comment:
yes, i saw that later and checked what they actually added. AdwinTrave  added correctly - he removed that instance of original code. 

thanks for noticing it!